src/components/DayPicker/DayPicker.tsx

```diff
@@ -122,12 +122,19 @@ export type DayPickerProps = Omit<TextFieldProps, 'value' | 'onChange'> & {
   value?: Date;
 
   /**
-   * A background color to use for the currently selected month.
+   * A background color to use for the currently selected day.
    *
    * Default is the main primary color of the theme.
    */
   selectedBackgroundColor?: string;
 
+  /**
+   * A text color to use for the currently selected day.
+   *
+   * Default is white.
+   */
+  selectedTextColor?: string;
```

Comment:
I think we should control this text color so we alway ensure it is accessible. On `palette` there is a helper function `getContrastText` which returns the accessible text color based on the passed in color (in this case the `selectedBackgroundColor`). If you want an example, I used this in `ColorPicker`.

Reply:
Not sure how this would work because `selectedBackgroundColor` can be any custom value and I believe `getContrastText` only works with `palette` colors? Or are you suggesting we do a breaking change and lock down `selectedBackgroundColor` to a `palette` color where we no longer allow custom values?

Reply:
Yeah the more I think about this, the more I think this goes against our normal patterns. In all of our components we limit the colors to primary or secondary palette colors. I would recommend we introduce a breaking change and replace `selectedBackgroundColor` with `color` ("primary" | "secondary" | "default"). 

I checked all of our implementations of DayPicker and none of them are passing in the `selectedBackgroundColor`. So this change should not cause any breaking changes with our current implementations. 

What are your thoughts?